ENIGMAsystem/SHELL/Platforms/xlib/BSD/BSDjoystick.cpp

```diff
@@ -25,20 +25,75 @@
 **                                                                              **
 \********************************************************************************/
 
-#include "../General/PFjoystick.h"
+#include "Platforms/General/PFjoystick.h"
+#include <string>
 
-namespace enigma {
-  void init_joysticks();
-  void handle_joysticks();
-}
+using std::string;
 
-namespace enigma_user
+namespace enigma
 {
+  void init_joysticks() {
+    enigma_user::joystick_load(0);
+    enigma_user::joystick_load(1);
+  }
 
-void joystick_map_button(int id, int butnum, char key);
-void joystick_map_axis(int id, int axisnum, char keyneg, char keypos);
+  void handle_joysticks() {
 
-extern int joystick_lastbutton;
+  }
+}  // namespace enigma
 
-}
+namespace enigma_user {
+
+  bool joystick_load(int id) {
```

Comment:
Try to avoid stubbing functions unless theyre required for compile and if they are please add DEBUG_MESSAGE informing users their game is using unimplemented functions